src/org/ods/scheduler/LeVADocumentScheduler.groovy

```diff
@@ -10,70 +10,70 @@ class LeVADocumentScheduler extends DocGenScheduler {
     // Document types per GAMP category
     private static Map GAMP_CATEGORIES = [
         "1": [
+            LeVADocumentUseCase.DocumentType.CFTP as String,
+            LeVADocumentUseCase.DocumentType.CFTR as String,
             LeVADocumentUseCase.DocumentType.CSD as String,
-            LeVADocumentUseCase.DocumentType.RA as String,
```

Comment:
I strongly vouch to leave the ordering as is .. (based on v-model) ...

Reply:
ok, will revert the order. was just easier to search through them that way/